src/guide/1-meta/1-philosophy.rst

```diff
@@ -1,2 +1,10 @@
 FLAMEGPU2 Design Philosophy
 ===========================
+
+FLAME GPU has the aim of accelerating the performance and scale of agent based simulation by targeting readily available parallelism in the form of Graphics Processing Units (GPUs). A central idea behind FLAME GPU is to abstract the GPU away from modellers so that modellers can build models without having to worry about writing parallel code. FLAME GPU also separates a model description from the model implementation. This simplifies the processes of validating and verifying models as the simulator code is tested in isolation from the model itself.
```

Comment:
modellers twice in quick succession, perhaps.

>... away from modellers so that model can be designed without having to worry about writing parallel code.

Designed could be replaced with implemented or something else.


---------------

> FLAME GPU also separates a model description from the model implementation.

This doesn't feel like an also, it's essentially repeating the "to abstract the GPU away from modellers"

Reply:
Is this ripped direct from the website? I recognise the below paragraphs.

Reply:
Yes just pulled from the website for now - want to discuss what should go here but this seemed like a decent starting point

Reply:
Yeah I've used bits of it on the rse site project for fgpu2, but just noticed the issue in first para.